app/models/train.rb

```diff
@@ -1,4 +1,12 @@
 class Train < ApplicationRecord
+  validates :name, presence: true
+  validates :total_seets, presence: true
+  validates :departure_time, presence: true
+  validates :arrival_time, presence: true
+  validates :price, presence: true
+  validates :direction_id, presence: true
```

Comment:
Why did you write validations again and again?  you can write validations in single line. If all this columns are in a same table.

Reply:
okay